pages/offices/[filename].tsx

```diff
@@ -103,32 +110,43 @@ const OfficeLayout = ({ office }) => {
           {`${office.streetAddress}, ${office.suburb}, ${office.addressRegion} ${office.postalCode}, ${office.addressCountry}`}
         </a>
       </h4>
-      <TinaMarkdown
-        components={componentRenderer}
-        content={office.directions}
-      />
+      <div data-tina-field={tinaField(office, "directions")}>
+        <TinaMarkdown
+          components={componentRenderer}
+          content={office.directions}
+        />
+      </div>
       {office.parking.children.length > 0 && (
         <>
           <h2>Parking</h2>
-          <TinaMarkdown
-            components={componentRenderer}
-            content={office.parking}
-          />
+          <div data-tina-field={tinaField(office, "parking")}>
+            <TinaMarkdown
+              components={componentRenderer}
+              content={office.parking}
+            />
+          </div>
         </>
       )}
       {office.publicTransport.children.length > 0 && (
         <>
           <h2>Public Transport</h2>
-          <TinaMarkdown
-            components={componentRenderer}
-            content={office.publicTransport}
-          />
+          <div data-tina-field={tinaField(office, "publicTransport")}>
+            <TinaMarkdown
+              components={componentRenderer}
+              content={office.publicTransport}
+            />
+          </div>
         </>
       )}
       {office.team.children.length > 0 ? (
         <>
-          <h2>The SSW {office.addressLocality} Team</h2>
-          <TinaMarkdown components={componentRenderer} content={office.team} />
+          <div data-tina-field={tinaField(office, "team")}>
+            <h2>The SSW {office.addressLocality} Team</h2>
+            <TinaMarkdown
+              components={componentRenderer}
+              content={office.team}
+            />
+          </div>
```

Comment:
The TinaField should only be on the editable part (i.e. the TinaMarkdown component), so probably better to wrap the TinaMarkdown in another element and put the Tina field on that - because at the moment its not consistent.